File: KoltinTestStore/KotlinTest2016/app/src/main/java/com/ethanco/kotlintest/_java/TestActivity.java
package com.ethanco.kotlintest._java;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import android.widget.Toast;

import com.ethanco.kotlintest.R;

/**
 * Created by Zhk on 2016/11/7.
 */

public class TestActivity extends AppCompatActivity {
    private TextView tvInfo;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnTest = (Button) findViewById(R.id.btn_test);
        tvInfo = (TextView) findViewById(R.id.tv_info);

        btnTest.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Toast.makeText(TestActivity.this, "Hello", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: KoltinTestStore/KotlinTest2016/app/src/main/java/com/ethanco/kotlintest/_kotlin/TestActivity.kt
package com.ethanco.kotlintest._kotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import com.ethanco.kotlintest.R

/**
 * Created by Zhk on 2016/11/7.
 */

class TestActivity : AppCompatActivity() {
    private var tvInfo: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnTest = findViewById(R.id.btn_test) as Button
        tvInfo = findViewById(R.id.tv_info) as TextView

        btnTest.setOnClickListener {
            //Toast.makeText(this@TestActivity, "Hello", Toast.LENGTH_SHORT).show()
            toast("Hello", Toast.LENGTH_SHORT)
        }
    }
}
